resources/manifests/pod-checkpointer-psp.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: pod-checkpointer-restricted
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default'
+spec:
+  privileged: false
+  # Required to prevent escalations to root.
+  allowPrivilegeEscalation: true
```

Comment:
To match the comment this should be `false` I believe. In any case, we're allowing to run as the root user below so I don't know if it matters...

Reply:
fixed.

Reply:
But if you do that, `securityContext` needs to be mutated, so it will pick up `privileged` policy again, no?

Reply:
I meant I fixed the comment and still allowed privilege escalation

Reply:
Ah, make sense :+1: